medusa/classes.py

```diff
@@ -132,6 +133,122 @@ def __init__(self, episodes=None, provider=None):
         # For example SearchType.MANUAL_SEARCH, SearchType.FORCED_SEARCH, SearchType.DAILY_SEARCH, SearchType.PROPER_SEARCH
         self.search_type = None
 
+        # # Assign a score for the amount of preffered words there are in the release name
+        self._preferred_words_score = None
+
+    @property
+    def preferred_words_score(self):
+        """Calculate a score based on the amount of preferred words in release name."""
+        preferred_words = [word.lower() for word in app.PREFERRED_WORDS]
+        self._preferred_words_score = round((len([word for word in preferred_words if word in self.name.lower()]) / float(len(preferred_words))) * 100)
+        return self._preferred_words_score
+
+    @staticmethod
+    def __qualities_to_string(qualities=None):
+        return ', '.join([Quality.qualityStrings[quality] for quality in qualities or []
+                          if quality and quality in Quality.qualityStrings]) or 'None'
+
+    def want_episode(self, season, episode, quality):
```

Comment:
@medariox i've moved want_episode from the Series class to Result class. Imo it looks allot cleaner, with less parameter. And it just feels like this is a better home.